corgi/core/models.py

```diff
@@ -586,6 +586,8 @@ class ProductStream(ProductModel):
     active = models.BooleanField(default=False)
     et_product_versions = fields.ArrayField(models.CharField(max_length=200), default=list)
 
+    exclude_components = models.JSONField(default=dict)
+
```

Comment:
Not familiar with the real prod-defs data, but based on the example in your test, can we use a simpler ArrayField(TextField) here instead? Or if the real data is more complicated / full JSON, let's make the test data match.

Reply:
just following status quo

Reply:
Not sure I understand - do we expect to receive full JSON values / dicts with subkeys from prod-defs, like `{"some_component": ["subcomponent_1", "subcomponent_2", "etc"]}`? In this case, let's make the test data match the real values we get.

Or are we only receiving simple lists of string component names, like `["component_1", "component_2", "etc"]`? In this case, let's use an ArrayField instead:
```suggestion
    exclude_components = fields.ArrayField(models.CharField(max_length=200), default=list)

```

Reply:
thx - useful conversation - changed it to use fields.ArrayField as you proposed with the caveat that hopefully some day we will have a 'plan' for product definition.